src/lib/badge/_badge-theme.scss

```diff
@@ -0,0 +1,165 @@
+@import '../core/theming/palette';
+@import '../core/theming/theming';
+@import '../core/typography/typography-utils';
+
```

Comment:
Add comment here that explains why the badge theme contains all of the styles for the badge instead of just color and typography